### PR TITLE
Update iotawatt.markdown for using the new high accuracy sensors

### PR DESCRIPTION
**Pull request by jyavenard**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [X] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
I feel like this PR has pending code changes?
Could you point me to the upstream PR?

**Comment by jyavenard** [comment]
https://github.com/home-assistant/core/pull/55512

this got merged a month ago.

**Comment by frenck** [comment]
Was that part of 2021.9?

**Comment by jyavenard** [comment]
> Was that part of 2021.9?

Nope.

**Comment by frenck** [comment]
In that case, this PR is targeting an incorrect branch, as the `current` branch is the current stable release, which is 2021.9.

Please change the target branch to `next` and perform a git rebase if needed.

**Comment by jyavenard** [comment]
hmmm this isn't what I wanted it to do :(

**Comment by jyavenard** [comment]
that's better.

File: source/_integrations/iotawatt.markdown [hunk 1]
```diff
@@ -20,9 +21,9 @@ and create them as sensors in Home Assistant.
 
 ## Energy management
 
-IoTaWatt does not provide the exact data that is needed for energy management. We're working with the IotaWatt team on resolving this.
+Starting with Home Assistant 2021.10 you can use the accumulated energy sensors directly with the energy management application.
```
Comment (frenck) [inline source/_integrations/iotawatt.markdown]: We don't keep a changelog in our docs. The docs always reflect the current state.

```suggestion
You can use the accumulated energy sensors directly with the Home Assistant energy dashboard.
```

File: source/_integrations/iotawatt.markdown [hunk 2]
```diff
@@ -35,18 +36,16 @@ You will need to configure two new IoTaWatt output sensors:
...
-    source: sensor.mainsexport
-    name: Total Grid Export
-    unit_prefix: k
-  - platform: integration
-    source: sensor.mainsconsumption
-    name: Total Grid Consumption
-    unit_prefix: k
-```
+#### If having a solar net system:
+The iotawatt team recommends that the inputs for solar reads positive which can be achieved by either changing the orientation of the CT sensor or in the IoTaWatt's input settings, check `Reverse`
```
Comment (frenck) [inline source/_integrations/iotawatt.markdown]:
```suggestion
#### Using a solar net system

The IoTaWatt team recommends that the inputs for solar reads positive which can be achieved by either changing the orientation of the CT sensor or in the IoTaWatt's input settings, check `Reverse`.
```

**Review by frenck — APPROVED** [review]
Thanks, @jyavenard 👍

**Pull request merged**